src/main/java/com/google/sps/utils/GameUtils.java

```diff
@@ -11,21 +11,22 @@
 // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 // See the License for the specific language governing permissions and
 // limitations under the License.
-
+ 
```

Comment:
There are a lot of lines with a single space added in this PR. If those are removed, it would make the PR much more readable and follow the style guides anyway.

Reply:
I think in all of these places a line was both added and removed (so like net 0 line); I think it was probably caused by something shifting?

Reply:
Is there a way to change the way this looks in the pr?

Reply:
You added a space to an otherwise empty line. As in the ` ` character you get when you press the spacebar.

Reply:
Oh shoot I totally missed that, yep. Those are taken out